src/js/engagement_view/node_modules/.gitignore

```diff
@@ -1,6 +0,0 @@
-# This exists to keep the existence of the node_modules directory in git.
```

Comment:
Probably best to keep this file in the repository for the time being; I'll be digging into it in my PR that'll follow this one.